QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,195 @@
+
+### License [3.iii]
+
+The license for Eclipse iceoryx is the Apache License 2.0, and all of the code includes a header stating that.
+The full license text is available in the [`LICENSE`](https://github.com/eclipse-iceoryx/iceoryx/blob/master/LICENSE) file.
+The project includes a [`NOTICE`](https://github.com/eclipse-iceoryx/iceoryx/blob/master/NOTICE.md) with links to more information about these licenses.
+
+There is some third-party content included with Eclipse iceoryx which is licensed as MIT or New BSD.
+Details can also be found in the included [`NOTICE`](https://github.com/eclipse-iceoryx/iceoryx/blob/master/NOTICE.md#third-party-dependencies) document.
+
+### Copyright Statement [3.iv]
+
+Each source code file in Eclipse iceoryx has a copyright header that needs to follow this [style](https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md#header).
+
+A CI job ensures by checking with `ament_copyright` that all files comply with this rule.
+
+## Testing [4]
+
+Every iceoryx package has a `test` folder that contains subfolders for [module -or integrationtests](https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md#testing) written based on the Google test Framework. All tests are running on the CI for every Pull-Request and merge to master for all supported platforms.
+Currently, due to limited support on Windows, there are some exclusions for testing on the CI.
+
+### Feature Testing [4.i]
+
+The features of iceoryx are tested by the [iceoryx_integrationtest](https://github.com/eclipse-iceoryx/iceoryx/tree/master/iceoryx_integrationtest) package. Using `launch_test` are tests on system-level executed to ensure that the iceoryx packages are functional and the Public API usable.
+There is continuous effort ongoing to cover the corner cases in the usage of iceoryx.
+
+### Public API Testing [4.ii]
+
+For every major feature all are tests executed. New features must provide unit and integration tests that cover the code changes in the Pull-Request. The tests reside in separated folders for every package following a defined structure and naming convention.
+The features are tested on module(unit) -integration and system test level. [Guidelines](https://iceoryx.io/latest/advanced/best-practice-for-testing/) for Contributors ensure high quality for test development.
+
+### Coverage [4.iii]
+
+The Eclipse iceoryx project use [gcov](https://gcc.gnu.org/onlinedocs/gcc/Gcov.html) to capture the following coverage types:
+
+- Line Coverage
+- Function Coverage
+- Branch Coverage
+
+The coverage results of every Pull-Request and master branch are publicly available on [codecov.io](https://app.codecov.io/gh/eclipse-iceoryx/iceoryx).
+If the user wants a more detailed report (e.g. the coverage in the different packages), he can follow [this](https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md#coverage-scan) guide to obtain them locally.
```

Comment:
a little bit more diversity `she can follow` :)

Reply:
Another possibility would be `one can follow` then we just assume consciousness or that some kind of entity is reading this.